### PR TITLE
Feature/profile edit

### PR DESCRIPTION
**Pull request by ue**

### CODE REVIEW

File: src/components/textInput/view/textInputView.js [hunk 1]
```diff
@@ -5,9 +5,9 @@ import { connect } from 'react-redux';
 // Styles
 import styles from './textInputStyles';
 
-const TextInputView = ({ isDarkTheme, innerRef, ...props }) => (
+const TextInputView = ({ isDarkTheme, innerRef, height, ...props }) => (
```
Comment (mistikk) [inline src/components/textInput/view/textInputView.js]: check above comment for props

File: src/constants/steemConnectOptions.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 export const steemConnectOptions = {
   base_url: 'https://app.steemconnect.com/',
   client_id: 'esteemapp',
-  redirect_uri: 'http://127.0.0.1:3415/', // http://127.0.0.1:3415
-  scope: 'vote,comment,delete_comment,comment_options,custom_json,claim_reward_balance,offline',
+  redirect_uri: 'http://127.0.0.1:3415/',
+  scope:
+    'vote,comment,delete_comment,comment_options,custom_json,claim_reward_balance,account_update,offline,account_update2',
```
Comment (mistikk) [inline src/constants/steemConnectOptions.js]: account_update2 ??

File: src/containers/profileEditContainer.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+      coverUrl: get(props.currentAccount, 'about.profile.cover_image'),
+      avatarUrl: get(props.currentAccount, 'avatar'),
+    };
+  }
+
+  // Component Life Cycles
+
+  // Component Functions
+
+  _handleOnSave = () => {};
```
Comment (mistikk) [inline src/containers/profileEditContainer.js]: ?

File: src/containers/profileEditContainer.js [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+        const _currentAccount = { ...currentAccount, display_name: name, avatar: avatarUrl };
+        _currentAccount.about.profile = { ...params };
+
+        dispatch(updateCurrentAccount(_currentAccount));
+
+        navigation.state.params.fetchUser();
+        navigation.goBack();
+      })
+      .catch(error => {
+        Alert.alert(get(error, 'message'));
```
Comment (mistikk) [inline src/containers/profileEditContainer.js]: Alert.alert(
            intl.formatMessage({
              id: 'alert.fail',
            }),
            error.message || error.toString(),
          );

File: src/providers/steem/dsteem.js [hunk 5]
```diff
@@ -232,7 +233,7 @@ export const ignoreUser = async (currentAccount, pin, data) => {
 
   if (currentAccount.local.authType === AUTH_TYPE.STEEM_CONNECT) {
     const token = decryptKey(currentAccount.local.accessToken, digitPinCode);
-    const api = new steemconnect.Client({
+    const api = new Steemconnect.Client({
```
Comment (mistikk) [inline src/providers/steem/dsteem.js]: i think it should be `steemconnect`

File: src/screens/application/container/applicationContainer.js [hunk 6]
```diff
@@ -487,13 +487,6 @@ class ApplicationContainer extends Component {
       const isExistUser = await getExistUser();
 
       realmObject[0].name = currentUsername;
-      dispatch(
-        updateCurrentAccount({
-          name: realmObject[0].username,
-          avatar: realmObject[0].avatar,
-          authType: realmObject[0].authType,
-        }),
-      );
```
Comment (mistikk) [inline src/screens/application/container/applicationContainer.js]: why did you remove this

Comment (ue) [reply src/screens/application/container/applicationContainer.js]: cuz its not necessary at all we already keeping on redux store
